api/base/serializers.py

```diff
@@ -143,92 +173,141 @@ def get_attribute(self, obj):
         return field_source_method(auth=auth)
 
 
-class JSONAPIHyperlinkedIdentityField(ser.HyperlinkedIdentityField):
+class RelationshipField(ser.HyperlinkedIdentityField):
```

Comment:
Is there a good reason to subclass `HyperlinkedIdentityField`? It doesn't seem there's much shared code with it. Also, it doesn't really make sense semantically for `RelationshipField` to be a `HyperlinkedIdentityField`.

From the docs for HyperlinkedIdentityField:

```
    A read-only field that represents the identity URL for an object, itself.

    This is in contrast to `HyperlinkedRelatedField` which represents the
    URL of relationships to other objects.
```

In our case, `RelationshipField` represents URLs of relationships to other objects, not the URL for the object being serialized.